src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,77 @@
+package duke.parser;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
```

Comment:
Perhaps 'import org.junit.jupiter.api.Test;` should be separated from the 'duke' import statements? Please do check if this is correct as I am not too sure on this as well.